articles/app-service/overview-managed-identity.md

```diff
@@ -443,9 +447,22 @@ For .NET applications and functions, the simplest way to work with a managed ide
     // OR
     var kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(azureServiceTokenProvider.KeyVaultTokenCallback));
     ```
+If you want to use a user-defined managed identity, then you'll need to use the following code instead modifying to target the right resources and use the right clientid:
+
+    ```csharp
+    using Microsoft.Azure.Services.AppAuthentication;
+    using Microsoft.Azure.KeyVault;
+    // ...
+    var azureServiceTokenProvider = new AzureServiceTokenProvider("RunAs=App;AppId=<clientid-guid>");
```

Comment:
Preference is to use environment variable configuration rather than hard coding.

Reply:
This thread can be resolved. I have created a suggestion comment capturing the change.